app-android/src/main/java/io/github/droidkaigi/confsched2022/KaigiApp.kt

```diff
@@ -203,8 +209,13 @@ fun KaigiAppDrawer(
             }
         }
     } else {
+        val keyboardController = LocalSoftwareKeyboardController.current
+        val drawerState = kaigiAppScaffoldState.drawerState
+        if (drawerState.isAnimationRunning && drawerState.isClosed) {
```

Comment:
It may not need to be strict this far🤔 
(It may be enough to judge hiding the software keyboard to use only `isAnimationRunning.`)

Reply:
I thinks this can be called a lot. How about using LaunchedEffect?

Reply:
Thank you for your advice!
I modified the following, could you please review it?
63310982d6e7f872b7c6c5fea91050bc474dca9d